definitions/npm/jest_v20.x.x/flow_v0.39.x-/test_jest-v20.x.x.js

```diff
@@ -227,11 +231,13 @@ expect(wrapper).toHaveHTML(true);
 
 expect(wrapper).toHaveProp("test");
 expect(wrapper).toHaveProp("test", "test");
-expect(wrapper).toHaveProp({ test: "test" });
 // $ExpectError
 expect(wrapper).toHaveProp();
 // $ExpectError
 expect(wrapper).toHaveProp(true);
+expect(wrapper).toHaveProp({ test: "test" });
+// $ExpectError
+expect(wrapper).toHaveProp({ test: "test" }, "test");
```

Comment:
Making this error is the reason for the complicated type signature, instead of just `toHaveProp(propKey: string | Object, propValue?: any): void`